src/context.ts

```diff
@@ -14,6 +14,8 @@ export const createExpressContext = (data, res) => {
 };
 
 export class Context {
+  models: any;
+  user: any;
```

Comment:
Is there a way to require this to be either an Object or null?

Reply:
Yes we can do that, will update in PR